app/models/stripe_dispute.rb

```diff
@@ -55,11 +55,12 @@ def serialize_on_update(input)
   end
   
   def fire_change_events
-    if changed? && !dispute&.is_legacy
-      if changed_attributes["object"].nil?
+    if saved_changes? && !dispute&.is_legacy
+      if after_save_changed_attributes["object"].nil?
```

Comment:
This is due to some changes in how after_commit callbacks work in Rails 5.1 and later. `changed?`,  `will_save_change_to_attribute`, `changed_attributes`  is used for the changes before the save happens, and now it's `saved_changes?`, `saved_change_to_attribute?`, and `saved_changes.transform_values(&:first)`, which I wrapped in a method called `after_save_changed_attributes`